lib/routes/universities/njupt/jwc.js

```diff
@@ -48,6 +48,15 @@ module.exports = async (ctx) => {
                     return Promise.resolve(JSON.parse(cache));
                 }
                 const response = await got.get(itemUrl);
+                if (response.redirectUrls.length !== 0) {
+                    const single = {
+                        title: titleList[index],
+                        link: itemUrl,
+                        description: '该通知无法直接预览, 请点击原文链接↑查看',
+                        pubDate: new Date(dateList[index]).toUTCString(),
+                    };
+                    return Promise.resolve(single);
```

Comment:
```suggestion
                    return {
                        title: titleList[index],
                        link: itemUrl,
                        description: '该通知无法直接预览, 请点击原文链接↑查看',
                        pubDate: parseDate(dateList[index]),
                    };
```
Please use parseDate() https://github.com/DIYgod/RSSHub/issues/6795
Promise.resolve() is not necessary https://github.com/DIYgod/RSSHub/pull/8028